README.md

```diff
@@ -1,26 +1,25 @@
 # Welcome to my Portfolio
 
-This project was made using React and Elastic UI as a learning practice.
+This project was made using [React](https://reactjs.org/) and [Elastic UI](https://elastic.github.io/) as a learning practice.
 
-This is the latest version is deployed here: www.beatrizsanchez.dev
+The latest version is deployed [here](https://www.beatrizsanchez.dev).
```

Comment:
It's unusual to include `www` in the addresses

Reply:
Updated